<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientChatWidget</class>
 <widget class="QWidget" name="ClientChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(54, 54, 54);
border-color: rgb(11, 11, 11);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="13,1">
   <item>
    <widget class="QListWidget" name="lstMessage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget#lstMessage{
background-image: url(:/image/fon.jpg)0 0 0 0 repeat repeat;
}</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="wdgSend" stretch="1,7,1">
     <item>
      <widget class="QLabel" name="lbMessage">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(54, 54, 54);</string>
       </property>
       <property name="text">
        <string>Message:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lnMessage">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(79, 79, 79);
border-radius: 8px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(70, 70, 70);
border-color: rgb(11, 11, 11);
color: rgb(255, 255, 255);
border-radius: 4px;</string>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
